ledcontroller/__init__.py

```diff
@@ -440,3 +453,26 @@ def batch_run(self, *commands):
                 args = command[1:]
                 cmd(*args)
         self.repeat_commands = original_retries
+
+
+
+def rgb_to_hue(red, green, blue):
```

Comment:
Yes. Just add a comment linking to the source. I don't see why that shouldn't be done for just a single function.